### DISCUSSION TITLE
请问这个工具可以用来做什么的？

### DISCUSSION THREAD

**Discussion by hongaah** [NelsonYong/react-echarts-json#3] [Q&A] [answered]
请问这个实际用途是什么，可以解决什么问题呀~

**Comment by NelsonYong** [comment, marked as answer]
不用改动代码即可完成图表数据样式的改变。举个简单的例子，UI说想看看某个柱状图数据增多的效果，如果没有这个工具你需要回到代码，将这个柱状图数据变更，然后代码重新跑才能看到效果，这无异于浪费了时间，但是这个工具可以实时的改，也不需要去改代码就可以随心所欲的改变图表

**Comment by NelsonYong** [comment]
还能在线编写代码实时改动图表，灵活性很高
